### ISSUE TITLE
Probably incorrect information in a markdown file

### ISSUE THREAD

**Issue by Youssef1313** [dotnet/roslyn#47084]
https://github.com/dotnet/roslyn/blob/92f024ab986153902d5aa15a2d079ccd28dd673d/docs/wiki/Troubleshooting-tips.md#verifying-the-version-of-the-compiler-and-language

> For environments where you cannot use the command-line, you can include #error version in your program and the compiler and language versions will be printed as an error message. (**Note this only works with C# 7.1 or later**)

I think this depends on the compiler version, not the language version. Maybe it's meant the compiler shipped with C# 7.1 or later?

**Comment by RikkiGibson** [comment]
Yeah, this probably implies Microsoft.CodeAnalysis 2.3 or 2.4, going by [this doc](https://github.com/dotnet/roslyn/blob/bf26b0607760eabf50ee20e60513b7492decea96/docs/wiki/NuGet-packages.md#versioning)
